src/stop.sh

```diff
@@ -104,7 +104,7 @@ do_umountall() {
     done
 
     #Get fuse mounts of the cluster
-    CEPH_FUSE_MNTS=$("${CEPH_BIN}"/ceph -c $conf_fn tell mds.* client ls 2>/dev/null | grep mount_point | tr -d '",' | awk '{print $2}')
+    CEPH_FUSE_MNTS=$(findmnt -t fuse.ceph-fuse -n --raw --output=source,target | awk '{print $2}')
```

Comment:
`--output=target`? No need of `awk`

Reply:
I thought about that, but then I forgot to change after testing :) Will update it.